changelogs/fragments/remove_deprecated.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Please refer to https://github.com/ansible-collections/cisco.nxos/pull/836/files#diff-044354a2e5989396b9168d5dde63bd5196447bfcf661637e7433555b532c50c6R1-R8 for changelog entry.